tests/functional/lib/BackbeatClient.js

```diff
@@ -83,4 +64,51 @@ describe('BackbeatClient unit tests with mock server', () => {
             return done();
         });
     });
+
+    it('should get bucket metadata', done => {
+        const destReq = backbeatClient.getBucketMetadata({
+            Bucket: bucketName,
+        });
+        return destReq.send((err, data) => {
+            assert.ifError(err);
+            assert.strictEqual(typeof data.acl, 'object');
```

Comment:
I suggest building the expected object using the json and assert using the `deepStrictEqual` method. This way you are validating type and value at the same time.